include/ps/kv_app.h

```diff
@@ -686,7 +686,7 @@ int KVWorker<Val>::AddPullCB(
       if (vals->empty()) {
         vals->resize(total_val);
       } else {
-        CHECK_EQ(vals->size(), total_val);
+        CHECK_GE(vals->size(), total_val);
```

Comment:
can we update the doc for Pull and ZPull for this change and the reliable way to read the valid msg size?